app/Http/Controllers/EventController.php

```diff
@@ -87,6 +87,11 @@ public function show(Request $request, Event $event)
                     'update' => optional($request->user())->can('update', $event) ?? false,
                 ],
             ],
+        ])->withViewData([
+            'meta' => [
+                'title' => $event->name,
+                'description' => strip_tags($event->html)
```

Comment:
Adding this worries me because I think the description meta tag is only suppose to be around 140 characters long. We could either cut the string down to 140 characters or just remove this tag - I'm not sure which way is best though.

Reply:
We can use Str::limit() (https://laravel.com/docs/6.x/helpers#method-str-limit)

Reply:
Ok let’s do that